### PR TITLE
Update component colors to primary

### PR DESCRIPTION
**Pull request by zoltanszogyenyi**
## 📑 Description

This PR fixes the remaining rogue blue colors to primary and also improves the documentation by fixing certain spacings, invalid heading/links in the docs and also updates the config file for the bg-primary-50 color from blue to orange.

## Status

- [x] Completed

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] I have checked the page with https://validator.unl.edu/
- [x] All the tests have passed
- [x] My pull request is based on the latest commit (not the npm version).

### CODE REVIEW

**Comment by shinokada** [comment]
@zoltanszogyenyi What do you think about [this proposal](https://github.com/themesberg/flowbite-svelte/issues/656)?

**Comment by zoltanszogyenyi** [comment]
> @zoltanszogyenyi What do you think about [this proposal](https://github.com/themesberg/flowbite-svelte/issues/656)?

I think that the current approach is good enough - style via classes from Tailwind CSS or props.

The fallback should be the `primary` from the Tailwind config file - and if anyone wants to change the colors they can do so by changing the config file or by overriding with Tailwind CSS classes.

**Comment by zoltanszogyenyi** [comment]
Hey @shinokada,

This is ready to be reviewed.

**Comment by shinokada** [comment]
Since you updated `exported` variables, can you run `pnpm gen:props` and push it, please? It will update all the props and format them.

File: src/routes/docs/pages/quickstart.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ const config = {
     extend: {
       colors: {
         // flowbite-svelte
-         primary: {50:'#ebf5ff',100: '#fff1ee',200:'#ffe4de',300:'#ffd5cc',400:'#ffbcad',500:'#fe795d',600:'#ef562f',700:'#eb4f27',800:'#d3330a',900:'#d3330a'}
+         primary: {50:'#fff1ee',100: '#fff1ee',200:'#ffe4de',300:'#ffd5cc',400:'#ffbcad',500:'#fe795d',600:'#ef562f',700:'#eb4f27',800:'#d3330a',900:'#d3330a'}
```
Comment (jjagielka) [inline src/routes/docs/pages/quickstart.md]: There's no difference between primary-50 and primary-100. As well primary-800 is the same as primary-900. Is that intentional?

File: src/routes/docs/components/skeleton.md [hunk 2]
```diff
@@ -183,7 +183,7 @@ The component has the following props, type, and default values. See <A href="/d
 ## References
 
 <P>
-  <A href="https://flowbite.com/docs/components/skeleton/" target="_blank" rel="noreferrer" class="link"
+  <A href="https://flowbite.com/docs/components/skeleton/" class="link"
```
Comment (jjagielka) [inline src/routes/docs/components/skeleton.md]: All those References should be changed to markdown as:
```[Tailwind CSS Skeleton](https://flowbite.com/docs/components/skeleton)```

File: src/routes/docs/pages/quickstart.md (hunk 1 above)
Comment (zoltanszogyenyi) [reply src/routes/docs/pages/quickstart.md]: You're right, it seems that in Figma it's the same color. @robert1508 can you check this please?

Screenshot 2023-06-04 at 19 02 46

File: src/routes/docs/components/skeleton.md (hunk 2 above)
Comment (zoltanszogyenyi) [reply src/routes/docs/components/skeleton.md]: @jjagielka this is how they were before, but sure, I can update them to use markdown

Comment (jjagielka) [reply src/routes/docs/components/skeleton.md]: It's not urgent. Just an observation. Using markdown will unify the look and feel in the long term though.

Comment (zoltanszogyenyi) [reply src/routes/docs/components/skeleton.md]: @jjagielka I've converted all of them to markdown + fixed some wrong text/links too

**Comment by zoltanszogyenyi** [comment]
> Since you updated `exported` variables, can you run `pnpm gen:props` and push it, please? It will update all the props and format them.

Running the command gives me the following error:

Screenshot 2023-06-04 at 19 04 19

**Comment by shinokada** [comment]
@zoltanszogyenyi Can you install all dependencies again?

```
pnpm I 
```

File: src/routes/docs/pages/quickstart.md (hunk 1 above)
Comment (robert1508) [reply src/routes/docs/pages/quickstart.md]: Everything is revised and I updated these colors, I added the color shades for primary-50, 800, and 900, and now everything is good to go.

**Comment by zoltanszogyenyi** [comment]
> @zoltanszogyenyi Can you install all dependencies again?
> 
> ```
> pnpm i
> ```

That worked, thanks! You can review the PR now. Also updated the primary colors based on @robert1508's updates on Figma @jjagielka.

**Review by shinokada — APPROVED** [review]
Thank you.

**Comment by shinokada** [comment]
Please use `docs:` instead of `docs(some content)`?
[Conventioal commit](https://www.conventionalcommits.org/en/v1.0.0/) has `fix:`, `feat:`, `chore:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, etc.

**Pull request merged**